pkg/report/predicate/vuln_test.go

```diff
@@ -0,0 +1,113 @@
+package predicate_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+	"time"
+
+	dbTypes "github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/aquasecurity/trivy/pkg/clock"
+	ftypes "github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/report/predicate"
+	"github.com/aquasecurity/trivy/pkg/types"
+)
+
+func TestWriter_Write(t *testing.T) {
+	testCases := []struct {
```

Comment:
It is a nitpick, but we're recently trying to comply with the same convention.
https://github.com/aquasecurity/trivy/blob/d93a9978005889638472ff198c4af6ef56b1246c/pkg/licensing/classifier_test.go#L16

```suggestion
	tests := []struct {
```

Reply:
I've fixed it.